rpc/wrtc_server_stream.go

```diff
@@ -268,6 +282,8 @@ func (s *webrtcServerStream) processHeaders(headers *webrtcpb.RequestHeaders) {
 }
 
 func (s *webrtcServerStream) processMessage(msg *webrtcpb.RequestMessage) {
+	s.logger.Debugw("incoming grpc request msg", "msg", s.msgCount)
```

Comment:
I don't think we should add these logs, maybe if we find a use case later on? but right now its just going to make debug output even more noisy without too much additional value @benjirewis